From 6670125e95f7c7e55b314fffec6084eb1a0cf718 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 14 Aug 2018 16:51:24 +0300
Subject: [PATCH] btl: remove SPL_OFFSET usage

SPL_OFFSET is not passed anymore, so free the scratchpad 28.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/btl/Makefile b/board/habanalabs/common/btl/Makefile
index 3dc91d4..1a3e880 100644
--- a/board/habanalabs/common/btl/Makefile
+++ b/board/habanalabs/common/btl/Makefile
@@ -29,12 +29,10 @@
 endif
 
 PRIMARY_CPP_FLAGS += $(COMMON_CPP_FLAGS)
-PRIMARY_CPP_FLAGS += -DSPL_OFFSET=0xa0000
 PRIMARY_CPP_FLAGS += -DSPL_SIZE=$(SPL_SIZE)
 PRIMARY_CPP_FLAGS += -DUBOOT_OFFSET=0xb0000
 
 SECONDARY_CPP_FLAGS += $(COMMON_CPP_FLAGS)
-SECONDARY_CPP_FLAGS += -DSPL_OFFSET=0x8a0000
 SECONDARY_CPP_FLAGS += -DSPL_SIZE=$(SPL_SIZE)
 SECONDARY_CPP_FLAGS += -DUBOOT_OFFSET=0x8b0000
 
diff --git a/board/habanalabs/common/btl/primary.S b/board/habanalabs/common/btl/primary.S
index 87b9da0..8e6f083 100644
--- a/board/habanalabs/common/btl/primary.S
+++ b/board/habanalabs/common/btl/primary.S
@@ -46,7 +46,6 @@
  * passing flash layout to CPU SW
  * ------------------------------
  */
-cfg mmPSOC_GLOBAL_CONF_SCRATCHPAD_28, SPL_OFFSET
 cfg mmPSOC_GLOBAL_CONF_SCRATCHPAD_29, SPL_SIZE
 cfg mmPSOC_GLOBAL_CONF_SCRATCHPAD_30, UBOOT_OFFSET
 cfg mmPSOC_GLOBAL_CONF_SCRATCHPAD_31, BTLID
